test/regression-extract.js

```diff
@@ -39,7 +43,7 @@ const extractTarGz = async (url, baseDir, include) => {
     next();
   });
   const response = await fetch(url);
-  await pipeline(response.body, extract);
+  await pipeline(response.body.pipe(zlib.createGunzip()), extract);
```

Comment:
Since this is a pipeline I think you can pass zlib.createGunzip() as a second argument instead of calling .pipe.

Reply:
Ahh, thanks for the heads-up! You're right.